src/cmd/keybaseca/keybaseca.go

```diff
@@ -157,8 +215,7 @@ func mainAction(c *cli.Context) error {
 			}(team)
 		}
 		semaphore.Wait()
-	}
-	if c.Bool("wipe-logs") {
+	} else if c.Bool("wipe-logs") {
```

Comment:
this will fail some formatters because it's not a switch statement. not a big deal, but it would probably be nicer as one.